### PR TITLE
fix/TA-671: fix dueDate task after creation in chat

### PR DESCRIPTION
**Pull request by aelbenney**
After creating a task in chat using `"++" `, the date is not shown. We need to add the `dueDate` to `options` in `message` in order to show it.

### CODE REVIEW

File: task-management/src/main/webapp/js/TaskChatPlugin.js [hunk 1]
```diff
@@ -88,6 +89,7 @@ var chatTaskComposerAppPlugin = {
     };
     this.saveTask(eXo.chat.userSettings, data).then(function(response) {return response.text()}).then(function(data) {
       data = JSON.parse(data);
+      message.options.dueDate = data.dueDate ? data.dueDate.split(" ")[0] : '';
```
Comment (thomasdelhomenie) [inline task-management/src/main/webapp/js/TaskChatPlugin.js]: This should be done in processData method, like for message.options.url and message.options.task.

File: task-management/src/main/webapp/js/TaskChatPlugin.js [hunk 2]
```diff
@@ -67,6 +67,7 @@ var chatTaskComposerAppPlugin = {
       options: {
         type: 'type-task',
         username: username,
+        dueDate: '',
```
Comment (thomasdelhomenie) [inline task-management/src/main/webapp/js/TaskChatPlugin.js]: Is this really needed ?

File: task-management/src/main/webapp/js/TaskChatPlugin.js (hunk 1 above)
Comment (thomasdelhomenie) [inline task-management/src/main/webapp/js/TaskChatPlugin.js]: This should be done in processData IMO, like message.options.url and message.options.task.

File: task-management/src/main/webapp/js/TaskChatPlugin.js (hunk 2 above)
Comment (thomasdelhomenie) [inline task-management/src/main/webapp/js/TaskChatPlugin.js]: Is this change really needed ?

File: task-management/src/main/webapp/js/TaskChatPlugin.js (hunk 1 above)
Comment (aelbenney) [reply task-management/src/main/webapp/js/TaskChatPlugin.js]: okay

File: task-management/src/main/webapp/js/TaskChatPlugin.js (hunk 2 above)
Comment (aelbenney) [reply task-management/src/main/webapp/js/TaskChatPlugin.js]: No, it's an old code !
